pkg/anypoint/types.go

```diff
@@ -92,6 +92,7 @@ type API struct {
 	Pinned                    bool     `json:"pinned"`
 	ProductVersion            string   `json:"productVersion"`
 	Tags                      []string `json:"tags"`
+	LastActiveDate            string   `json:lastActiveDate`
```

Comment:
Is this field still needed after your new changes?

Reply:
No, i can remove it.